TRANSLATION.md

```diff
@@ -12,20 +12,20 @@ Suggestion on words and terms:
 
 | Original word/term | Suggestion |
 | ------------------ | ---------- |
-| arrow function | *arrow function* |
+| arrow function | función flecha |
 | bug | error |
 | Bundler | *Bundler* |
 | camelCase | *camelCase* |
 | callback | *callback* |
 | controlled component | *controlled component* |
 | debugging | depuración |
 | library | biblioteca |
-| lowercase | *lowercase* |
+| lowercase | minúscula(s) |
 | props | *props* |
 | React element | Elemento de React |
-| state | *state* |
-| string | *string* |
-| template literals | *template literals* |
+| state | estado |
+| string | cadena de texto |
```

Comment:
I think `string` should not be changed, it is a widely used term across the industry.

Reply:
When you start learning about programming (and I think this applies to any language) you learn primitives, and with primitives comes `string`, `boolean`, `number`...etc.

That said, if someone reads `string` they would know that we're talking about `cadena de texto`.

Reply:
:+1: to keep string without translate it

Reply:
I think cadena de caracteres works too.

> > How can I translate `array` and `keys` ?
> 
> Array could be translated as `arreglo` en Spanish, but while there is people who use it (usually un universities) I think array is a commonly used term in Spanish.
> 
> Key are `llaves`, but again is something used as is in Spanish too.

I think `keys` as in `keys` of an `object`, could be `propiedades` de un `objeto`. `arreglos` have `'índices` that are also `propiedades` when seen as `objetos`.